backend/mail/locales/security-notifications.en.yaml

```diff
@@ -0,0 +1,48 @@
+password_update_text:
+  description: ""
+  other: "The password for your {{ .ServiceName }} account has been updated."
+subject_password_update:
+  description: ""
+  other: "Your password has been updated"
+
+primary_email_update_text:
+  description: ""
+  other: "The primary email address for your {{ .ServiceName }} account has been updated from {{ .OldEmailAddress }} to {{ .NewEmailAddress }}."
+subject_primary_email_update:
+  description: ""
+  other: "Your primary email address has been updated"
+
+email_create_text:
+  description: ""
+  other: "The email address {{ .NewEmailAddress }} has been added to your {{ .ServiceName }} account."
+subject_email_create:
+  description: ""
+  other: "A new email address has been registered"
+
+email_delete_text:
+  description: ""
+  other: "The email address {{ .DeletedEmailAddress }} has been removed from your {{ .ServiceName }} account."
+subject_email_delete:
+  description: ""
+  other: "An email address has been removed"
+
+passkey_create_text:
+  description: ""
+  other: "Your {{ .ServiceName }} account has been registered with a new passkey."
```

Comment:
I also get this message when I already have an account registered and I add a passkey through the profile, so it is not the account that has been registered but the passkey itself (see also the `other` value in the subject, which feels like a better fit for this).